docs/src/creating-packages.md

```diff
@@ -620,6 +620,11 @@ may fit your package better.
        library, even without explicit affiliation with the creators of the OpenSSL (provided
        there's no copyright or trademark infringement etc.)
 
+9. Packages should follow the [Stylistic Conventions](https://docs.julialang.org/en/v1/manual/variables/#Stylistic-Conventions).
+     * The package name begin with a capital letter and word separation is shown with upper camel case
+     * Packages that provide the functionality of a project from another language should use the Julia convention
+     * Packages that [provide pre-built libraries and executables](https://docs.binarybuilder.org/stable/jll/) can keep orignal name, but should get `_jll`as a suffix. For example `pandoc_jll` wraps pandoc.
```

Comment:
```suggestion
     * Packages that [provide pre-built libraries and executables](https://docs.binarybuilder.org/stable/jll/) can keep orignal name, but should get `_jll`as a suffix. For example `pandoc_jll` wraps pandoc. But note that the generation and release of most JLL packages are handled by the [Yggdrasil](https://github.com/JuliaPackaging/Yggdrasil) system. 
```

Reply:
Very good addition! Is a sentence starting with “but” grammatically correct? We could also just leave out the “but” then.

Reply:
Updated to `However`